apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -64,6 +64,12 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
     return `/documents?${params.toString()}`;
   };
 
+  const documentStatuses = [
```

Comment:
I've dealt with this in the `feat/inbox` branch, please revert this file for now 💚

Reply:
Done 💯